### PR TITLE
[Marketplace] LPS-189777 Create solution template for Clarity solution trial 

### PR DESCRIPTION
**Pull request by ytuik**
https://liferay.atlassian.net/browse/LPS-189777

### CODE REVIEW

**Comment by ytuik** [comment]
ci:test:sf

**Comment by ytuik** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:1]: Sort the properties of CSS in ascending order.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/configuration.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+					"label": "Publisher Email",
+					"name": "publisherEmail",
+					"type": "text"
+				},
+				{
+					"description": "11111",
+					"label": "Product ID",
+					"name": "productID",
+					"type": "text"
+				}
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/configuration.json:16]: The name of this property would be camel case.
```suggestion
				{
					"description": "11111",
					"label": "Product ID",
					"name": "productId",
					"type": "text"
				}
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+.banner {
+  background-color: #F7F7F8;
+  width: 80rem;
+  height: 7.5rem;
+  border-radius: 1rem;
+  box-shadow: 0 0.3125rem 0.9375rem 0 rgba(19, 20, 31, 0.12);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:10]: Please use bootstrap here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  margin: 0 0 12px 0;
+}
+
+.banner__liferay-container {
+  margin-right: 1.5rem;
+}
+
+.banner__title-container {
+  display: flex;
+  flex-direction: column;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:39]: same here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.banner__info-container {
+  display: flex;
+  align-items: center;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:44]: same here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.banner__share-button {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:69]: same here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+.hero-container__description{
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  flex-direction: column;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css:7]: Same here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css:1]: Sort the properties of CSS in ascending order.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

**Review by welitonpe — CHANGES_REQUESTED** [review]
Needs little adjusts, but good work  :fire:

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/slider-marketplace/index.js [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+function activateItem(activeItem, nextItem, movement) {
+	activeItem.classList.remove('active');
+	nextItem.classList.add('active');
+
+	if (movement) {
+		activeItem.classList.remove(movement);
+		nextItem.classList.remove(movement);
+	}
+}
```
[the remark this answers is not in the document]
Comment (ytuik) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/slider-marketplace/index.js:53]: Hi, i didn't build this fragment, it's a liferay default one that i copied to change some style props to match figma, so i didn't really look how it works
so to make a change in the logic i will need some time to understand the code, please let me know if this change is necessary

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/slider-marketplace/index.js [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+function move(movement, index = null) {
```
[the remark this answers is not in the document]
Comment (ytuik) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/slider-marketplace/index.js:55]: same thing of the comment bellow

**Comment by ytuik** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thanks @ytuik :white_check_mark:

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Review by welitonpe — APPROVED** [review]
Good Job! :fire: :fire:

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

**Comment by ytuik** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
Good job @ytuik :martial_arts_uniform:

**Comment by ytuik** [comment]
ci:forward

**Comment by ytuik** [comment]
ci:test:site-initializer-liferay-marketplace:nocompile

**Comment by ytuik** [comment]
ci:forward

**Pull request closed without merging**